src/main/java/com/cinema/shop/Main.java

```diff
@@ -66,7 +66,14 @@ public static void main(String[] args) throws AuthenticationException {
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
         authenticationService.register("chester@mail.com", "1234f");
-        log.info(authenticationService.login("chester@mail.com", "1234f"));
+        log.info("Trying to log in user: "
+                + authenticationService.login("chester@mail.com", "1234f"));
```

Comment:
remove this and remove `throws AuthenticationException` from method signature

Reply:
done, and logs config changed (does't show config changes maybe because it's in git ignore), log::info - should be also changed?